packages/grid/_modules_/grid/components/default-footer.tsx

```diff
@@ -25,23 +25,20 @@ export const DefaultFooter = React.forwardRef<HTMLDivElement, DefaultFooterProps
       return null;
     }
 
-    const isPaginationAvailable = !!paginationComponent;
-    const showRowCount = !options.hideFooterRowCount && !isPaginationAvailable && (
-      <RowCount rowCount={totalRowCount} />
-    );
-    const showSelectedRowCount = !options.hideFooterSelectedRowCount && (
-      <SelectedRowCount selectedRowCount={selectedRowCount} />
-    );
-    const justifyItemsEnd = !selectedRowCount && !options.hideFooterSelectedRowCount;
+    const showSelectedRowCount =
+      !options.hideFooterSelectedRowCount && selectedRowCount > 0 ? (
```

Comment:
`selectedRowCount > 0 ` I don't think that should be in the footer as it's a `SelectedRowCount` concern.

The footer should be dumb. 
ie if not hidden show...

Reply:
I doubt that the footer can be dumb, it needs to manage the space available

Reply:
well it should just show hide the underlying component but it doesn't need to know about the business logic within it. ie 
Show selected rows if screen less than Xpx => footer responsibility
Show selected rows for counter above 0 is the selected rows business logic

Reply:
> well it should just show hide the underlying component but it doesn't need to know about the business logic within it. ie

How would you solve the positioning problem in this case? Mind that I'm using `selectedRowCount` here, and @DanailH did too in the current version:

https://github.com/mui-org/material-ui-x/blob/325d91116efe567df4e563e6eef7b696e3629190/packages/grid/_modules_/grid/components/default-footer.tsx#L35

Reply:
I agree, I see the footer more like a manager and the components that are in it should be fairly dumb. Unfortunately the footer is the one responsible for the layout.